packages/rx_bloc_cli/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## [5.1.1]
```

Comment:
Please use the semantic versioning. 
- `5.1.1` communicates that a bugfix has been distributed.
- `5.2.0` communicates that a feature has been distributed.

Reply:
Done